frontend/src/app/modules/matches/matches-computations.js

```diff
@@ -17,9 +17,15 @@ export const generateMatchRatingChanges = (userId, userMatches) => userMatches.m
     }
 })
 
-export const generateRatingHistoryGraphForUser = (userMatches, userId, initialRating) => userMatches
-  .reverse()
-  .map(
+export const generateRatingHistoryGraphForUser = (userMatches, userId, initialRating) => {
+  const reversedUserMatches = userMatches.reverse()
+  const firstMatchDate = userMatches[0] ? userMatches[0].date : null
```

Comment:
theoretically could be rewritten to const firstMatchDate = userMatches[0] && userMatches[0].date, 
but don't know how much this is just a matter of taste so feel free to leave it like it is